docs/src/main/sphinx/release/release-445.md

```diff
@@ -0,0 +1,68 @@
+# Release 445 (17 Apr 2024)
+
+## General
+
+* Add support for large constant arrays. ({issue}`21566`)
+* Add the `query.dispatcher-query-pool-size` configuration property to prevent
+  the coordinator from hanging when too many queries are being executed at once. ({issue}`20817`)
+* Improve performance of queries selecting only catalog, schema, and relation
+  name from the `system.metadata.materialized_views` table. ({issue}`21448`)
+* {{breaking}} Remove the deprecated `legacy.materialized-view-grace-period`
+  configuration property. ({issue}`21474`)
+* Increase the number of columns supported by `MERGE` queries before failing
+  with a `MethodTooLargeException` error. ({issue}`21299`)
+* Fix potential query hang when there is an error processing data. ({issue}`21397`)
+* Fix possible worker crashes when running aggregation queries due to
+  out-of-memory error. ({issue}`21425`)
+* Fix incorrect results when performing aggregations over null values. ({issue}`21457`)
+* Fix query failure when using types that cannot be compared as part of a
+  `WHERE` clause. ({issue}`21508`)
+* Fix incorrect results for distinct count aggregations over a constant value. ({issue}`18562`)
+* Fix sporadic query failure when filesystem caching is enabled. ({issue}`21342`)
+* Fix unexpected failure for join queries containing predicates that might raise
+  an error for some inputs. ({issue}`21521`)
+
+## BigQuery connector
+
+* Add support for reading materialized views with the BigQuery storage API. ({issue}`21487`)
```

Comment:
Remove "with the BigQuery storage API". That's an implementation detail that's not relevant to users.